app/jobs/get_usps_waiting_proofing_results_job.rb

```diff
@@ -0,0 +1,40 @@
+class GetUspsWaitingProofingResultsJob < GetUspsProofingResultsJob
+  queue_as :long_running
+
+  def perform(_now)
+    return true unless ipp_enabled? && ipp_ready_job_enabled?
+
+    @enrollment_outcomes = {
+      enrollments_checked: 0,
+      enrollments_errored: 0,
+      enrollments_expired: 0,
+      enrollments_failed: 0,
+      enrollments_in_progress: 0,
+      enrollments_passed: 0,
+    }
+
+    reprocess_delay_minutes = IdentityConfig.store.
+      get_usps_proofing_results_job_reprocess_delay_minutes
+    enrollments = InPersonEnrollment.needs_status_check_on_waiting_enrollments(
+      ...reprocess_delay_minutes.minutes.ago,
+    )
+
+    started_at = Time.zone.now
+    analytics.idv_in_person_usps_proofing_results_job_started(
+      enrollments_count: enrollments.count,
+      reprocess_delay_minutes: reprocess_delay_minutes,
+      job_name: self.class.name,
```

Comment:
So @NavaTim my thinking is that we'll want some insight into which job handles which enrollments in case of any issues. Would you suggest a different approach? I could pass in job name as a separate field.

Reply:
I think it makes sense to use field that's separate from the event name, similar to how you have `job_name` here.

Reply:
@NavaTim  I totally agree.